test/snippet/range/view/interleave.cpp

```diff
@@ -0,0 +1,22 @@
+//![general]
+#include <range/v3/algorithm/equal.hpp>
+
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/view/interleave.hpp>
+
+
+int main()
+{
+std::string u{"FOOBARBAXBAT"};
+std::string i{"in"};
+size_t s = 3;
+
+// pipe notation
+auto v0 = u | seqan3::view::interleave(s, i);
+// or, via constructor.
```

Comment:
```suggestion
// or, via function notation
```
(it's not a constructor strictly speaking; you invoke the operator() on the functor and then that invokes a constructor)